docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,122 @@
+
+## Step 2: Install the Tiller server
+
+Tiller is the server-side component of Helm that runs inside your Kubernetes cluster. To install the Tiller server to the cluster you should use the `helm` client installed by the previous step. However, before we do so, for RBAC enabled clusters we need to create a service account for the Tiller server.
+
+For the purposes of this guide we'll create a Kubernetes service account with super-user cluster access (`cluster-admin`). Check out the guide on [Tiller and Role-Based Access Control](https://github.com/helm/helm/blob/master/docs/rbac.md) to learn about restricting the scope of Tiller in the cluster.
+
+Begin by creating a file named `rbac-tiller.yaml` with the following content:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+```
+
+Create the service account using the `kubectl` command tool:
+
+```bash
+kubectl create -f rbac-tiller.yaml
+```
+
+Next, we use the `helm` client to install the Tiller server to the cluster:
+
+```bash
+helm init --service-account tiller
+```
+
+Voila, you have installed Helm to your cluster.
+
+## Step 3: Install Kubeapps
+
+Kubeapps is a open-source web-based UI for deploying and managing applications in Kubernetes clusters. It provides you a dashboard to:
+
+    - Browse Helm charts from public or your own private chart repositories and deploy them to your cluster
+    - Upgrade, manage and delete the applications that are deployed in your cluster
+    - Browse and provision external services from the Service Catalog
+
+Kubeapps can be installed using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps) from the official [Bitnami charts repository](https://github.com/bitnami/charts).
+
+Begin by configuring the Helm client to use the Bitnami charts repository.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+Now you can easily install Kubeapps to your Kubernetes cluster with the following command:
+
+```bash
+helm install --name kubeapps --namespace kubeapps bitnami/kubeapps \
+    --set mongodb.metrics.enabled=true \
+    --set ingress.enabled=true \
+    --set ingress.hosts[0].name=kubeapps.[YOUR-BKPR-ZONE] \
+    --set ingress.hosts[0].tls=true \
+    --set ingress.hosts[0].tlsSecret=kubeapps-tls \
+    --set ingress.hosts[0].certManager=true
+```
+
+The command line flags provided in the above command enable the Ingress resource in the Kubeapps chart and also enable TLS support. Once installed, you will be able to access the Kubeapps dashboard securely (HTTPS) over the internet at `https://kubeapps.[YOUR-BKPR-ZONE]`.
```

Comment:
Minor nit, but you say "Once installed, you will be able to access the Kubeapps dashboard securely (HTTPS)...". I would qualify that: "Once installed, you will be able to generate an access token and then access the Kubeapps dashboard securely (HTTPS) over the Internet at `https://kubeapps.[YOUR-BKPR-ZONE]`"